packages/ilios-common/app/styles/ilios-common/components/course/visualizations.scss

```diff
@@ -0,0 +1,16 @@
+@use "../../mixins" as m;
+
+.course-visualizations {
```

Comment:
Is this a new consolidation? I couldn't find the matching file in the list, but make it's buried in deletes from other places?

Reply:
yes. was `packages/ilios-common/app/styles/ilios-common/components/course-visualizations.scss` before.